docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst

```diff
@@ -3,16 +3,58 @@
 How to validate data without a Checkpoint
 =========================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide demonstrates how to load an Expectation Suite and validate data without using a :ref:`Checkpoint<how_to_guides__validation__how_to_create_a_new_checkpoint>`. This might be suitable for environments or workflows where a user does not want to or cannot create a Checkpoint.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
 
+  - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+  - :ref:`Created an Expectation Suite <how_to_guides__creating_and_editing_expectations>`
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+The following code mirrors the code provided in the ``validation_playground.ipynb`` notebooks in ``great_expectations/notebooks``. First of all, we import Great Expectations, load our :ref:`Data Context<data_context>`, and define variables for the Datasource and Expectation Suite we want to access:
```

Comment:
```suggestion
The following code mirrors the code provided in the ``validation_playground.ipynb`` notebooks in ``great_expectations/notebooks``. First of all, we import Great Expectations, load our :ref:`Data Context<data_context>`, and define variables for the Datasource we want to access:
```

We don't actually do the Exp Suite until much later, so I wouldn't mention it yet.

Reply:
Good catch!